src/pages/app/stories/SearchBar.stories.svelte

```diff
@@ -0,0 +1,58 @@
+<script context="module">
+  import { Story, Template } from "@storybook/addon-svelte-csf";
+  import SearchBar from "../SearchBar.svelte";
+  import * as mock from "./mock.js";
+
+  export const meta = {
+    title: "App / Search / Search Bar",
+    component: SearchBar,
+    parameters: { layout: "fullscreen" },
+  };
+</script>
+
+<Template let:args>
+  <div class="container">
+    <SearchBar {...args} />
+  </div>
+</Template>
+
+<Story
+  name="default"
+  args={{ search: "project:example-project-123" }}
+  parameters={{
+    msw: {
+      handlers: [mock.users.data, mock.organizations.data, mock.projects.data],
```

Comment:
User and organization mocking is working as expected, but typing `project:` doesn't trigger any network requests.

Reply:
This seems to also be an issue in production, so let's consider it a bug to be fixed.